trestle/tasks/csv_to_oscal_cd.py

```diff
@@ -576,14 +576,27 @@ def _get_control_implementation(
         component.control_implementations.append(control_implementation)
         return control_implementation
 
+    def _str_to_list(self, value: str) -> List[str]:
+        """List to string."""
+        rval = []
+        if ',' in value:
+            values = value.split(',')
+            # remove whitespace
+            for v in values:
+                n = v.replace(' ', '')
+                n = n.strip()
```

Comment:
What is the purpose of both the lines above? Just to remove leading and trailing spaces, or also remove spaces in between? Suppose a single value is a string containing 2 or more words. This will remove spaces in between words also. Is that the intention?

Reply:
Fixed.